src/mixins/amalgamation-mixin.ts

```diff
@@ -294,17 +294,17 @@ export default class AmalgamationMixin extends Vue {
    */
   async refetchAmalgamatingBusinessesInfo (): Promise<AmalgamatingBusinessIF> {
     const fetchTingInfo = async (item: any): Promise<AmalgamatingBusinessIF> => {
-      const tingBusiness = await this.fetchAmalgamatingBusinessInfo(item.identifier)
-      // no auth info and business info means foreign, otherwise LEAR (affiliated or non-affiliated)
-      if (!tingBusiness.authInfo && !tingBusiness.businessInfo) {
+      // check if foreign
+      if (item.foreignJurisdiction) {
```

Comment:
I think I like this.

Can you remove the comment on line 301 now?

Reply:
sure

Reply:
Karim, this should work, right?